src/gui/CemuApp.cpp

```diff
@@ -176,6 +180,10 @@ bool CemuApp::OnInit()
 	SetTopWindow(m_mainFrame);
 	m_mainFrame->Show();
 
+#if BOOST_OS_LINUX && HAS_WAYLAND
+	wxWlSetAppId(m_mainFrame, "info.cemu.Cemu");
```

Comment:
HAS_WAYLAND only indicates compile time support for Wayland, the actual window still might end up using X11. I recommend adding an extra check like this:
```cpp
if (wxWlIsWaylandWindow(m_mainFrame))
    wxWlSetAppId(m_mainFrame, "info.cemu.Cemu");
```
And then implement wxWlIsWaylandWindow like this:
```cpp
bool wxWlIsWaylandWindow(wxFrame* frame)
{
    GtkWidget* gtkWindow = static_cast<GtkWidget*>(frame->GetHandle());
    GdkWindow* gdkWindow = gtk_widget_get_window(gtkWindow);

    return GDK_IS_WAYLAND_WINDOW(gdkWindow);
}
```
Note that this code is untested and I wrote it off the top of my head. You could also avoid the extra function and check inside `wxWlSetAppId` directly, but personally I think conditionally calling wxWlSetAppId makes it more obvious that the window isn't always wayland baked.